script/gen_requirements_all.py

```diff
@@ -32,7 +32,6 @@
     'i2csense',
     'credstash',
     'bme680',
-    'homekit',
```

Comment:
Have the extra dependencies for homekit been removed?
See note here:
https://www.home-assistant.io/components/homekit_controller/

I think that was why we had commented it here.

Reply:
Indeed thats out of date, those are no longer required! They were worked around so I could turn on tests here and to make it easier to install on mac / raspbian.

Reply:
(I'll figure out the docs repository tomorrow and try and submit a PR)

Reply:
PR: https://github.com/home-assistant/home-assistant.io/pull/8307